BolajiOlajide/a_socials
# Ensure that the "load more" button is visible on the events page

#### What Does This PR Do?
This PR ensures that the load more button is visible when there are more events to be shown.

#### Description Of Task To Be Completed
- Ensure that when there are no events to be shown the "load more" button is not visible
- Ensure that when there are more events to be shown then the "load more" button becomes visible 

#### How can this be manually tested?
- Ensure docker is installed locally on machine
- Ensure the environment variables in .env.sample are added in .env file in the root directory
- To build and spin up the docker containers, run make build
- To spin up the containers at a different time after building, run make start

#### What are the relevant github issues?
[#161058875](https://www.pivotaltracker.com/story/show/161058875)

#### Screenshot(s)
<img width="1440" alt="Screenshot 2019-04-09 at 8 54 12 AM" src="https://user-images.githubusercontent.com/25434963/55782920-3530b900-5aa5-11e9-80fe-1f3d1e6db014.png">
<img width="1440" alt="Screenshot 2019-04-09 at 8 50 51 AM" src="https://user-images.githubusercontent.com/25434963/55782773-e84ce280-5aa4-11e9-8368-1da48ae64a5c.png">
<img width="1440" alt="Screenshot 2019-04-09 at 8 52 35 AM" src="https://user-images.githubusercontent.com/25434963/55782816-00bcfd00-5aa5-11e9-9b28-c5d6c6601a63.png">
